js-envs/src/main/scala/org/scalajs/jsenv/JSEnv.scala

```diff
@@ -9,12 +9,43 @@
 
 package org.scalajs.jsenv
 
-import org.scalajs.core.tools.io._
-import org.scalajs.core.tools.classpath._
-import org.scalajs.core.tools.logging._
+import org.scalajs.core.tools.io.VirtualJSFile
+import org.scalajs.core.tools.jsdep.ResolvedJSDependency
+import org.scalajs.core.tools.linker.LinkingUnit
 
 trait JSEnv {
   /** Prepare a runner for the code in the virtual file. */
-  def jsRunner(classpath: CompleteClasspath, code: VirtualJSFile,
-      logger: Logger, console: JSConsole): JSRunner
+  def jsRunner(libs: Seq[ResolvedJSDependency], sjsCode: VirtualJSFile,
+      code: VirtualJSFile): JSRunner
+
+  def loadLibs(libs: Seq[ResolvedJSDependency],
+      sjsCode: VirtualJSFile): LibJSEnv = {
+    new LibJSEnv {
+      def unloadLibs: JSEnv = JSEnv.this
```

Comment:
Even more obscure is `unloadLibs`. It seems to be used only in two places both doing something like `jsEnv.unloadLibs.getClass`. Why not something more explicit like a `def description: String` method?